.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: File a bug report
+labels: ["Type: Bug", "Status: Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to fill out this bug report! Before submitting your issue, please make
+        sure you are using the latest version of the charm. If not, please switch to this image prior to 
+        posting your report to make sure it's not already solved.
```

Comment:
What does "switch to this image" mean in the context of a machine charm? Perhaps a better reading here would be
```
If not, please switch to the latest commit prior to posting....
```

Reply:
Updated in https://github.com/omnivector-solutions/slurmdbd-operator/pull/1/commits/9f31f333d28e0e0891f8231e156f2be43a72fecc.

Rather than latest commit I used latest version. I decided to use version instead because commits may break functionally (especially if you are going through an API ungrade). Version encourages end-users to be pulling the latest, officially released version of the charm. Let me know what you think!

Reply:
nice mate, good work